### ISSUE TITLE
wrong field after join?

### ISSUE THREAD

**Issue by aborruso** [saulpw/visidata#964]
Hi,
I have these two tables:

| sesso | domicilioCom | domicilioProv |
| --- | --- | --- |
| M | 082053 | PA |
| M | 082070 | PA |
| M | 082053 | PA |
| M | 082014 | PA |

| cod_istat | comune |
| --- | --- |
| 082014 | Caccamo |
| 082053 | Palermo |
| 082070 | Termini Imerese |
| 082071 | Terrasini |

If I run inner join, using `domicilioCom` and `cod_istat`, I have

|domicilioCom  |cod\_istat |sesso  |domicilioProv  |comune           |
|--------------|-----------|-------|---------------|-----------------|
|082053        |M          |M      |PA             |Palermo          |
|082053        |M          |M      |PA             |Palermo          |
|082070        |M          |M      |PA             |Termini Imerese  |
|082014        |M          |M      |PA             |Caccamo          |

Why I have `cod_istat` with those values, why is it a `sesso` duplicate?

I'm using `saul.pw/VisiData v2.5dev`

Thank you

```
sesso,domicilioCom,domicilioProv
M,082053,PA
M,082070,PA
M,082053,PA
M,082014,PA

cod_istat,comune
082014,Caccamo
082053,Palermo
082070,Termini Imerese
082071,Terrasini
```

**Comment by saulpw** [comment]
Hi @aborruso, thanks for filing this issue.  The join was happening correctly, but the key  columns were incorrect if they were named differently.  It should be fixed in develop now and will be in v2.5.

**Comment by aborruso** [comment]
Thank you very much
